examples/basic-audio.html

```diff
@@ -123,10 +128,16 @@
           }
         ]
       });
-
       // Called by Networked-Aframe when connected to server
       function onConnect () {
         console.log("onConnect", new Date());
+
+        // Handle mic button click (Mute and Unmute)
+        micBtnEle.addEventListener('click', function() {
+          NAF.connection.adapter.enableMicrophone(!micEnabled);
+          micEnabled = !micEnabled;
+          micBtnEle.innerHTML = micEnabled ? 'Mic Mute' : 'Mic Unmute';
```

Comment:
Prefer `.textContent` instead of `.innerHTML` when you know it's just text. `.innerHTML` may be restricted by CSP, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/require-trusted-types-for

Reply:
I would have put the words the other way around.
Mic Mute -> Mute Mic
Mic Unmute -> Unmute Mic
What do you think?

Reply:
Both changes Done